.github/workflows/test-examples-subprocess.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04]
-        python: ["3.8", "3.9", "3.10", "3.11"]
+        python: ["3.10", "3.11", "3.12"]
```

Comment:
Following @guilhem-barthes' [comment](https://github.com/Substra/substra-documentation/pull/430/files#r1689976102), python 3.9 has been removed here

Reply:
It was intentional, since I realised that the documentation CI is still running in 3.8 as of today despite Python 3.8 support being dropped since March, and I was afraid that this would happen again for 3.9. 
But I can also add a line in the Linear card instead and keep the CI running on 3.9 until that happens if you think it's better :)

Reply:
Fair enough, looks ok remove IMO!